core/app/core/src/lib/fields/dynamic-field/dynamic-field.component.html

```diff
@@ -54,16 +54,18 @@
     </a>
 </ng-container>
 <ng-container *ngIf="!isLink() && !hasOnClick()">
-    <ndc-dynamic
-        [ndcDynamicComponent]="componentType"
-        [ndcDynamicInputs]="{
+    <div [title]="field.definition.help ?? '' ">
```

Comment:
Hi @yunusyerli1, better to use a `<span>` instead of a div. Fields can be used inline and in many places, not only on the standard views

Reply:
Done.